gridt/src/app/movements/movements-detail/movements-detail.page.spec.ts

```diff
@@ -1,30 +1,38 @@
-import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { RouterTestingModule } from '@angular/router/testing';
-import { HttpClientModule } from '@angular/common/http';
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { RouterTestingModule } from "@angular/router/testing";
+import { HttpClientModule } from "@angular/common/http";
+<<<<<<< HEAD
+=======
 
-import { MovementsDetailPage } from './movements-detail.page';
-import { AuthService } from 'src/app/core/auth.service';
-import { of } from 'rxjs';
+>>>>>>> fe1feac8813acb9cbd3248540d796f9397e807e6
```

Comment:
I don't think you meant to have this here. Did you have issues with git?

Reply:
Yes, i did indeed have some trouble.